src/firmware/app/control/trajectory_planner_test.cpp

```diff
@@ -897,6 +1039,61 @@ TEST_F(TrajectoryPlannerTest, test_get_constant_time_interpolation_curved_line)
                 const_interp_trajectory.trajectory_elements[0].time, 0.001);
 }
 
+TEST_F(TrajectoryPlannerTest, test_get_constant_time_interpolation_too_many_elements)
+{
+    Polynomial2dOrder3_t path = {
+        .x = {.coefficients = {2, 0, 1, 0}},
+        .y = {.coefficients = {1, 0, 1, 0}},
+
+    };
+
+    FirmwareRobotPathParameters_t path_parameters;
```

Comment:
Can you describe _how_ you created a scenario that results in too many elements?

Also I may have missed it, but would be good to explain what that means / why we check it somewhere (not necessarily here).

Reply:
Added comment explaining 👍